### ISSUE TITLE
Godot change #includes -> absolute path

### ISSUE THREAD

**Issue by Ranoller** [sanja-sa/gddragonbones#24]
Please someone test my branch with godot 3.0 / 2.1.  If it compiles i do the pull request to stay up of date, i don´t want break backwards compatibility but i don´t use godot 3.0 - 2.1 jet.  

https://github.com/Ranoller/gddragonbones

**Comment by PiOverFour** [comment]
Hi, I didn't see this issue and just re-did the same fix... There is a way to keep compatibility, using the `__has_include()` preprocessor directive. [From this answer](https://stackoverflow.com/a/33260104/4561348), it's compatible with Clang, GCC >= 5.X and Visual Studio >= VS2015 Update2.

[I pushed a commit](https://github.com/PiOverFour/gddragonbones/commit/3710e8514af57ca64e7ce59fab46e59bdfd4ad39) to my fork using this. I works fine for Godot 2.1 and 3.1, with GCC. It may not work however for Godot 3.0...

**Comment by Ranoller** [comment]
Great.... there are two solution jet. If someone is using 2.1 3.0 have more options... but maybe this module should branch in the same way that godot... 
...2.1 will not be updated anymore so is not necesary to update this plugin for this branch and will be focused in 3.1... @sanja-sa will say.

**Comment by PiOverFour** [comment]
I agree that it may be better to leave a specific branch for 2.1 backward compatibility.
I'm actually trying to make the properties more consistent with the Godot 3 GDscript API, and it already breaks backward compat... Well, as you say, sanja-sa will decide!

**Comment by realkotob** [comment]
I tried @Ranoller's fork and it worked with latest godot master commit.

Edit: Tried two export options from Dragon Bones, JSON and Binary (dbbin), and they both work.

**Comment by Ranoller** [comment]
Great, but i do not pretend people use my fork, it's only a commit above and a commit that breaks compatibility. This solution should be integrated in the original repository and i ask to someone using other godot versions for test compatibility and decide proper solution to compatibility breaking (branch, continue with if indef, deeprecated godot 2/3 etc...) I think that will be convenient to start a branch for godot 3.1 and leave the photo to godot 2 and 3

**Comment by realkotob** [comment]
First of all thank you for being a lifesaver.

Godot 3.1 will be launched soon so I think the master branch should focus on that, and have branches for 2 and 3.0 instead. 

Very very few people will be interested in the older versions anyway, 99% of traffic will just want the latest version.

Also disregard what I said about about JSON option not working, I tried again and it also worked like DBBIN did.
